fastlane/Fastfile

```diff
@@ -176,8 +214,8 @@ end
     cocoapods()
 
     build_app(
-      workspace: "Simplenote.xcworkspace", 
-      scheme: "Simplenote",
+      workspace: workspace,
+      scheme: internal_scheme,
       clean: true,
       output_directory: "./build/")
```

Comment:
Combine some [Han Solo](https://github.com/woocommerce/woocommerce-android/pull/3592#pullrequestreview-592833189) DRY'ing + me playing human rubocop, and you get:
```suggestion
      output_directory: BUILD_FOLDER
    )
```